prov/gni/include/gnix_ep.h

```diff
@@ -416,6 +416,42 @@ int gnix_pep_bind(fid_t fid, struct fid *bfid, uint64_t flags);
  */
 ssize_t gnix_cancel(fid_t fid, void *context);
 
+/**
+ * Get an endpoint option
+ *
+ * @param[in] fid       the endpoint
+ * @param[in] level     the option level, must be FI_OPT_ENDPOINT
+ * @param[in] optname   the option name
+ * @param[out] optval   the value of the named option
+ * @param[out] optlen   the width of the value in bytes
+ *
+ * @return FI_SUCCESS	upon successfully returning the value of the option
+ * @return -FI_EINVAL   missing or invalid fid, optval, and/or optlen
+ * @return -FI_ENOENT   unsupported context type
+ * @return -FI_ENOPROTOOPT  unknown option level or name
+ */
+
+int gnix_getopt(fid_t fid, int level, int optname,
```

Comment:
why did you change the function name from ```gnix_ep_getopt``` to ```gnix_getopt```?  we've usually used a qualifier after the ```gnix``` string to indicate what type of libfab object the function operates on.

Reply:
I followed the model used for gnix_cancel.  When getopt is called, there are two separate paths taken, one for regular 'ep' endpoints and one for scalable 'sep' endpoints.  The gnix_ep_getopt() name was retained for use with a regular endpoint, and the gnix_sep_getopt() name was used for a scalable endpoint.   Like gnix_cancel, I then used gnix_getopt() for the routine which decides which subroutine to call.  Does this make sense?

Reply:
Duh, there is no gnix_sep_getopt() routine, I'm not doing well today.   But I did follow the gnix_cancel model.

The alternative would be to change the name to gnix_ep_getopt, and then the routine that it calls would have to be something like __gnix_ep_getopt() I guess.   Is that what you'd like?

Reply:
hmm.. i see your point about the routine names.  so let's stick with what you have.  Could you update the copyright though?

Reply:
Done.